client/ayon_core/hosts/maya/api/pipeline.py

```diff
@@ -67,6 +67,9 @@
 
 AVALON_CONTAINERS = ":AVALON_CONTAINERS"
 
+# Track whether the workfile tool is about to save
+ABOUT_TO_SAVE = False
```

Comment:
Already merged, but this looks like public constant. But the value is changing. Can this be changed to private, not capitalized variable `_about_to_save`?

Reply:
Will make a PR in a bit that does this for all hosts - same logic is also in Fusion and Houdini